src/Vlingo.Wire.Tests/Vlingo.Wire.Tests.csproj

```diff
@@ -18,6 +18,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Update="vlingo-actors.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
     <None Update="vlingo-actors.properties">
```

Comment:
Why do you keep those ?

Reply:
It stuck there, I'll update

Reply:
There is no properties.cs file, wire only use the actors configuration. At least for now?